README.md

```diff
@@ -1,40 +1,159 @@
-# ONNX Models
+# Open Neural Network eXchange (ONNX) Model Zoo
 
-This is a repository for storing ONNX models.
+[![Generic badge](https://img.shields.io/badge/Status-Work_In_Progress-red.svg)](#) 
+[![Generic badge](https://img.shields.io/badge/Contributions-Welcome-brightgreen.svg)](contribute.md)
 
-### Models
+<p align="center">
+	<img src="images/ONNX Model Zoo Graphics.png" width="60%"/>
```

Comment:
please remove the empty space in the file name, thanks

Reply:
updated to ONNX_Model_Zoo_Graphics.png